documentation/src/main/asciidoc/user_guide/endpoint_interop.adoc

```diff
@@ -1,54 +1,77 @@
 [[endpoint_interop]]
-== Embedded/Remote Interoperability
+== Protocol Interoperability
 
-{brandname} offers the possibility to store and retrieve data in a local embedded way, and also remotely thanks to multiple endpoints offered.
+Clients exchange data with {brandname} through endpoints such as REST or Hot Rod.
 
-When configuring the cache with the link:#encoding_media_type[MediaType] of the data to be stored, users can read and write data in multiple formats taking advantage of the internal support for data conversion.
+Each endpoint uses a different protocol so that clients can read and write data in the cache in a suitable storage format. Because {brandname} can interoperate with multiple clients at the same time, it must convert data between client formats and the storage formats.
```

Comment:
Suggestion: change to "Each endpoint uses a different protocol so that clients can read and write data in a suitable storage format"